solutions/observability/get-started/quickstarts.md

```diff
@@ -31,7 +31,7 @@ Follow the steps in these guides to get started quickly:
 Want to use {{fleet}} or some other feature not covered in the quickstarts? Follow the steps in these guides to get started:
 
 * [Get started with system metrics](/solutions/observability/infra-and-hosts/get-started-with-system-metrics.md).
-* [Get started with application traces and APM](/solutions/observability/apm/get-started-fleet-managed-apm-server.md).
+* [Get started with application traces and APM](/reference/fleet/get-started-managed-apm-server.md).
```

Comment:
Same thing here. As a new user, I don't have any context of what Fleet-managed APM Server means. So it feels like I'm being sent to the wrong page because of the mismatch between link text on this page and the H1 of the page I'm sent to.

Reply:
Something to remember when reorganizing content at this scale. Thanks, I'll edit the title as well. I rescued the previous diagram and intro for Fleet-managed APM Server BTW.